src/Components/EnergyCalculator/Steps/Utilities.tsx

```diff
@@ -0,0 +1,100 @@
+import { FormattedMessage } from 'react-intl';
+import QuestionHeader from '../../QuestionComponents/QuestionHeader';
+import QuestionQuestion from '../../QuestionComponents/QuestionQuestion';
+import QuestionDescription from '../../QuestionComponents/QuestionDescription';
+import RHFOptionCardGroup from '../../RHFComponents/RHFOptionCardGroup';
+import { z } from 'zod';
+import { zodResolver } from '@hookform/resolvers/zod';
+import { useForm, SubmitHandler } from 'react-hook-form';
+import { useParams } from 'react-router-dom';
+import { ReactComponent as Plug } from '../Icons/Plug.svg';
+import { ReactComponent as LowFuel } from '../Icons/LowFuel.svg';
+import PrevAndContinueButtons from '../../PrevAndContinueButtons/PrevAndContinueButtons';
+import { useDefaultBackNavigationFunction, useGoToNextStep } from '../../QuestionComponents/questionHooks';
+import { useContext } from 'react';
+import { Context } from '../../Wrapper/Wrapper';
+import useScreenApi from '../../../Assets/updateScreen';
+
+const Utilities = () => {
+  const { formData, setFormData } = useContext(Context);
```

Comment:
I created a hook to automatically add the energy calculator data if it is not there. Also, if `energyDataAvailable` is `true` the TS knows that `formData.energyCalculator` is not undefined. You can use it to fix the TS errors.
```suggestion
  const { formData, setFormData } = useContext(Context);
  const energyDataAvailable = useEnergyFormData(formData);
```